### PR TITLE
Add nscale feature

### PR DESCRIPTION
**Pull request by nbassler**

### CODE REVIEW

File: pymchelper/readers/shieldhit.py [hunk 1]
```diff
@@ -231,15 +234,19 @@ def read_payload(self, detector):
...
         # normalize result if we need that.
         if detector.dettyp not in (SHDetType.dlet, SHDetType.tlet,
+                                   SHDetType.letflu,
+                                   SHDetType.dletg, SHDetType.tletg,
                                    SHDetType.avg_energy, SHDetType.avg_beta,
                                    SHDetType.material):
             detector.data /= np.float64(detector.nstat)
-
+            if scale != 1:  # scale with number of particles given by user
+                detector.data *= nscale
```
Comment (nbassler) [inline pymchelper/readers/shieldhit.py]: missing: MeV/g -> Gy

File: pymchelper/detector.py [hunk 2]
```diff
@@ -263,7 +264,8 @@ def merge_list(input_file_list,
 def merge_many(input_file_list,
                conv_names=(SHConverters.standard.name,),
                nan=False,
-               colormap=SHImageWriter.default_colormap):
+               colormap=SHImageWriter.default_colormap
```
Comment (grzanka) [inline pymchelper/detector.py]: There is missing comma (,) here

File: pymchelper/readers/shieldhit.py [hunk 3]
```diff
@@ -231,15 +235,28 @@ def read_payload(self, detector):
...
+                                   SHDetType.dletg, SHDetType.tletg,
                                    SHDetType.avg_energy, SHDetType.avg_beta,
                                    SHDetType.material):
             detector.data /= np.float64(detector.nstat)
 
+        if nscale != 1 and detector.dettyp in (SHDetType.energy, SHDetType.fluence, SHDetType.crossflu,
+                                               SHDetType.dose, SHDetType.counter, SHDetType.pet):
+            detector.data *= np.float64(nscale)  # scale with number of particles given by user
+            if detector.dettyp == SHDetType.dose:
+                # 1 megaelectron volt / gram = 1.60217662 x 10-10 Gy
```
Comment (nbassler) [inline pymchelper/readers/shieldhit.py]: confirmed.

File: pymchelper/readers/shieldhit.py [hunk 4]
```diff
@@ -179,10 +179,12 @@ def get_detector_unit(detector_type, geotyp):
...
         """
         if geotyp == SHGeoType.zone:
-            dose_units = (" MeV/primary", "Dose*volume")
+            dose_units = ("MeV/primary", "Dose*volume")
+            dose_gy_units = ("Gy", "Dose*volume")
             alanine_units = ("MeV/primary", "Alanine RE*Dose*volume")
         else:
             dose_units = (" MeV/g/primary", "Dose")
+            dose_gy_units = ("Gy", "Dose")
             alanine_units = ("MeV/g/primary", "Alanine RE*Dose")
```
Comment (nbassler) [inline pymchelper/readers/shieldhit.py]: -> Gy

File: pymchelper/readers/shieldhit.py [hunk 5]
```diff
@@ -179,10 +179,12 @@ def get_detector_unit(detector_type, geotyp):
         :return:
         """
         if geotyp == SHGeoType.zone:
-            dose_units = (" MeV/primary", "Dose*volume")
+            dose_units = ("MeV/primary", "Dose*volume")
+            dose_gy_units = ("Gy", "Dose*volume")
             alanine_units = ("MeV/primary", "Alanine RE*Dose*volume")
```
Comment (nbassler) [inline pymchelper/readers/shieldhit.py]: -> J

File: pymchelper/run.py [hunk 6]
```diff
@@ -63,9 +64,10 @@ def main(args=sys.argv[1:]):
         parsed_args.outputfile = files[0][:-3] + "txt"
 
     if parsed_args.many:
-        merge_many(files, parsed_args.converter, parsed_args.nan, parsed_args.colormap)
+        merge_many(files, parsed_args.converter, parsed_args.nan, parsed_args.colormap, parsed_args.nscale)
```
Comment (grzanka) [inline pymchelper/run.py]: I agree it can be done nicer, but not in the scope of this PR

**Comment by grzanka** [comment]
Fixes #102

**Pull request merged**